.circleci/config.base.yml

```diff
@@ -358,6 +358,20 @@ jobs:
             bash ./.circleci/publish.sh
       - run: *scan_e2e_test_artifacts
 
+  post_deploy:
+    <<: *linux-e2e-executor
+    steps:
+      - attach_workspace:
+          at: ./
+      - restore_cache:
+          keys:
+            - amplify-category-api-ssh-deps-{{ .Branch }}
+      - install_github_cli
+      - run:
+          name: Run Post Publish Tasks
+          command: |
+            chmod +x .circleci/post_publish.sh && ./.circleci/post_publish.sh
```

Comment:
Why not `chmod +x` and commit change to git so that this doesn't need to be done on circleci?

Reply:
That should work too, let me do that.